crates/redis_interface/src/lib.rs

```diff
@@ -35,9 +35,57 @@ pub struct RedisConnectionPool {
     pub pool: fred::pool::RedisPool,
     config: RedisConfig,
     join_handles: Vec<fred::types::ConnectHandle>,
+    subscriber: RedisClient,
+    publisher: RedisClient,
     pub is_redis_available: Arc<atomic::AtomicBool>,
 }
 
+pub struct RedisClient {
+    inner: fred::prelude::RedisClient,
```

Comment:
I think we can have a deref trait implemented for this to avoid accessng the client through inner all the time.